service/controller/resource/prepareawscniformigration/create.go

```diff
@@ -109,8 +110,8 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		r.logger.Debugf(ctx, "Daemonset %q doesn't have routes-fixer container", dsName)
 		ds.Spec.Template.Spec.Containers = append(ds.Spec.Template.Spec.Containers, corev1.Container{
 			Name:    "routes-fixer",
-			Image:   ds.Spec.Template.Spec.Containers[0].Image,
-			Command: []string{"/usr/bin/bash"},
+			Image:   fmt.Sprintf("%s/giantswarm/alpine:3.17.3", r.registryDomain),
```

Comment:
Not sure if we really want to hard set the image version of alpine here. 
Is there any other way to get the version somewhere?

Reply:
would you rather have this as an helm chart value?

Reply:
Do we need to bump it regularly, if so helm chart value might make sense?

Reply:
No worries, let's merge it.